lib/fs.js

```diff
@@ -385,6 +385,13 @@ fs.readFile = function(path, options, callback) {
   if (!nullCheck(path, callback))
     return;
 
+  // doing justice between linux and windows
+  // for `/` char at the end of file path
+  if (isWindows && typeof path === 'string' &&
+    path.charAt(path.length - 1) === '/') {
```

Comment:
`path.charAt()` on this line should be aligned with `isWindows` on the previous line.